internal/cloud/awscloud/maintenance_test.go

```diff
@@ -0,0 +1,31 @@
+package awscloud_test
+
+import (
+	"testing"
+
+	ec2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/osbuild/osbuild-composer/internal/cloud/awscloud"
+)
+
+func TestEC2RemoveSnapshotAndDeregisterImage(t *testing.T) {
```

Comment:
a test for the 2h timing is missing, right?
just to know if the calculation is correct for sure

Reply:
Added a test to filter the reservations properly.